### DISCUSSION TITLE
pro

### DISCUSSION THREAD

**Discussion by giscus** [mathieucarbou/YaSolR#17] [Website comments]
# pro

Pro

https://yasolr.carbou.me/pro

**Comment by benoit-cty** [comment]
Bravo !

Excellent work : easy to install and configure with your great documentation !

Just a point to know for others : when you change MQTT config, you have to deactivate and reactivate the main MQTT option to make it works.

Do you plan to make a version to support remote RoboDyn ? I'd like to put the robodyn under the heater with a second ESP32 to also measure the water temperature.

**Reply by mathieucarbou** [reply]
Thanks, I will update the doc. Please note that the project is not finished yet: mostly the automatic dimmer mode with routing algorithms are still to implement, even if you can for now manually control the dimmers. All the "Work" icons in the front page identify the features in progress or to come.
Yes I plan to do the remote features, I hope in the next months and I hope I will be able to use the ESP Now feature in order to not add additional hardware to the mix. For that feature, ESP32 boards supporting a pigtail connector for an external antenna will be better suited.

**Reply by mathieucarbou** [reply]
Oh! And.... Thanks a lot for the support! I've just seen your donation ;-)

**Comment by erix22** [comment]
Woaw, great work !
as you said on the bottom of this page "Please consider buying the Pro"
I am ready for it, where can I do that please ?

**Reply by mathieucarbou** [reply]
Thanks a lot!
Project is still in development ;-) I will try to implement a really dumb automatic routing PID for the moment until I have the time to rewrite a good dimmer lib. Once it is done I will make it available.

**Comment by erix22** [comment]
Thank you so much for your answer and your time.
Good talk


Le lun. 3 juin 2024 à 19:19, Mathieu Carbou ***@***.***> a
écrit :

**Comment by libussa** [comment]
hello!
I sent some support your way, can I get access to the nice(r) pro UI please :)

**Reply by erix22** [reply]
??

**Reply by mathieucarbou** [reply]
@libussa it is done :-) thanks for the support and sorry for the delay, I usually process within the same day, but got distracted the past days.

**Reply by libussa** [reply]
@mathieucarbou no worries, and thanks! I however noticed you gave me push access to the repo, that's probably not what you meant to do?

**Reply by mathieucarbou** [reply]
@libussa no worry: this repo just serves a storage for firmware ;-) And nobody can push: there are some rulsets in force.
